understanding/20/focus-order.html

```diff
@@ -54,6 +54,28 @@ <h2>Intent of Focus Order</h2>
          Web page. Care should be taken so that the focus order makes sense to both of these
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
+
+      <p>Focus order also does not need to follow the visual presentation of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, visually the assumption in western left-to-right systems may be that elements in the
+         left-hand column would receive focus first. However, if meaning and operation are not affected,
+         it is not a failure if elements in the right-hand column receive focus first.
+         Again, as a best practice, care should be taken to make sure the focus order does not
```

Comment:
Suggest: "As a best practice, make sure the focus order does not..."

Reply:
Done